matminer/featurizers/site.py

```diff
@@ -481,15 +506,23 @@ def featurize(self, struct, idx):
                                             range(1, len(neigh_sites) + 1)])[0]
                         opval = opval or 0  # handles None
                         cn_fingerprint.append(wt * opval)
-
-        return cn_fingerprint
+        chem_fingerprint = []
+        if self.chem_info is not None:
+            for val in absdeltas.values():
+                chem_fingerprint.append(val / Nwt)
+        return cn_fingerprint+chem_fingerprint
 
     def feature_labels(self):
         labels = []
-        for cn in sorted(self.op_types):
-            for op in self.op_types[cn]:
-                labels.append("{} CN_{}".format(op, cn))
-
+        max_cn = sorted(self.op_types)[-1]
+        for k in range(max_cn):
+            cn = k + 1
+            if cn in list(self.ops.keys()):
+                for op in self.op_types[cn]:
+                    labels.append("{} CN_{}".format(op, cn))
+        if self.chem_info is not None:
+            for prop in self.chem_props:
+                labels.append("cn-wt-weighted av. abs. diff {}".format(prop))
```

Comment:
this title is too long, it's going to get cut off in the table headers and if you have multiple properties, you will only see "cn-wt-weighted" (and nothing else) for every property.

Maybe "{} local abs. diff" (?)